games-strategy/wesnoth/wesnoth-1.16.10.recipe

```diff
@@ -0,0 +1,112 @@
+SUMMARY="Turn-based tactical strategy game with a high fantasy theme"
+DESCRIPTION="Build up a great army, gradually turning raw recruits into \
+hardened veterans. In later games, recall your toughest warriors and form a \
+deadly host whom none can stand against! Choose units from a large pool of \
+specialists, and hand-pick a force with the right strengths to fight well on \
+different terrains against all manner of opposition.
+
+Wesnoth has many different sagas waiting to be played. Fight to regain the \
+throne of Wesnoth, of which you are the legitimate heir... step into the \
+boots of a young officer sent to guard a not-so-sleepy frontier outpost... \
+lead a brutal quest to unite the orcish tribes... vanquish a horde of undead \
+warriors unleashed by a foul necromancer, who also happens to have taken your \
+brother hostage... guide a band of elvish survivors in an epic quest to find \
+a new home... get revenge on the orcs by using the foul art of necromancy...
+
+There are at least two hundred unit types, sixteen races, six major factions, \
+and hundreds of years of history. The world of Wesnoth is absolutely huge and \
+only limited by your creativity — make your own custom units, create your own \
+maps, and write your own scenarios or even full-blown campaigns. You can also \
+challenge up to eight friends—or strangers—and fight in epic multiplayer \
+fantasy battles."
+HOMEPAGE="http://wesnoth.org/"
+COPYRIGHT="2003-2023 The Battle for Wesnoth"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://sourceforge.net/projects/wesnoth/files/wesnoth-1.16/wesnoth-$portVersion/wesnoth-$portVersion.tar.bz2"
+CHECKSUM_SHA256="a60521b1ee1c2ddc7dfde89a038b893a4b43337085bd1478f4bc92dab7da7fc7"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	wesnoth$secondaryArchSuffix
+	cmd:wesnoth
+	cmd:wesnothd
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	icu73$secondaryArchSuffix
+	lib:libboost_filesystem$secondaryArchSuffix
+	lib:libboost_iostreams$secondaryArchSuffix
+	lib:libboost_locale$secondaryArchSuffix
+	lib:libboost_program_options$secondaryArchSuffix
+	lib:libboost_regex$secondaryArchSuffix
+	lib:libboost_system$secondaryArchSuffix
+	lib:libboost_thread$secondaryArchSuffix
+	lib:libcairo$secondaryArchSuffix
+	lib:libfontconfig$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libfribidi$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgobject_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libintl$secondaryArchSuffix
+	lib:libpangocairo_1.0$secondaryArchSuffix
+	lib:libsdl_2.0$secondaryArchSuffix
```

Comment:
libSDL here and libSDL2 in devel?
These should be lib:libSDL2_2.0 see issue https://github.com/haikuports/haikuports/issues/3878